pkg/controller/controller.go

```diff
@@ -34,7 +34,7 @@ type Controller struct {
 
 // New creates an instance of the service catalog Controller.
 func New(w *watch.Watcher) (*Controller, error) {
-	h, err := createHandler(storage.CreateTPRStorage(w))
+	h, err := createHandler(tpr.CreateStorage(w))
```

Comment:
actually, I think it should be `NewStorage` so that usage of the package reads `tpr.NewStorage()`. This way, there can be other functionality inside the packages (`tpr` and `mem`)